src/state/index.js

```diff
@@ -29,6 +30,7 @@ export const rootReducer = combineReducers({
   location: locationReducer,
   serviceRating: serviceRatingReducer,
   gender: genderReducer,
+  serviceTypeProgramsReducer: serviceTypeProgramsReducer,
```

Comment:
I would rename this to 'serviceType'

Reply:
Oh I see, get rid of the reducer on the key.